server.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const helmet = require("helmet");
+const morgan = require("morgan");
+
+const projectsRouter = require("./projects/projectsRouter.js");
+const actionsRouter = require("./actions/actionsRouter.js");
+
+const server = express();
+
+server.use(express.json());
+server.use(helmet());
+server.use(morgan("dev"));
+
+server.use("/api/projects", projectsRouter);
+server.use("/api/actions", actionsRouter);
+
```

Comment:
Working server.

But it could work better with a sanity check for your '/' endpoint!